docs/source/cn/_toctree.yml

```diff
@@ -0,0 +1,74 @@
+- title: "Get started"  #"入门指南"
+  sections:
+    - local: index
+      title: 主页
+    - local: quick-start
+      title: 快速开始
+    - local: installation
+      title: 安装
+- title: "How-to guides"  #"操作指南"
```

Comment:
I think that we do need to translate the title so that they can show up in Chinese.